example.yaml

```diff
@@ -53,3 +53,17 @@ experience:
       - Developer
     short: One sentence description NASA
     more: more information here...
+
+  - title: Odd job during studies, experience not added to total experience
+    customer: Burger King
+    period:
+      excludefromtotalexp: true
```

Comment:
I guess it's not the period as such that you want to exclude; rather this project from the period calculation.
Is there perhaps a better term(attribute) on the project we can use over having a specific use-case attribute
in the period? then we can keep the period as only a definition of a period between two dates.
Maybe something like
`Project.Temporary bool`
Default would be false indicating it was a full time job which works in a backwards compatible way.